docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -299,6 +299,294 @@ When I scan barcode from context and save result to scenario variable `qrCodeLin
 Then `${qrCodeLink}` is equal to `https://www.example.com`
 ----
 
+=== Element Steps
+
+=== Upload file
+
+This step uploads a file with the given relative path.
+
+NOTE: A relative path starts from some given working directory, avoiding the need to provide the full absolute path (i.e. 'about.jpeg' for the files placed in the src/main/resources directory or '/story/uploadfiles/about.png')
+
+[source,gherkin]
+----
+When I select element located by `$locator` and upload `$resourceNameOrFilePath`
+----
+
+_Deprecated syntax (will be removed in VIVIDUS 0.7.0)_:
+[source,gherkin]
+----
+When I select element located `$locator` and upload file `$filePath`
+----
+
+* `$locator` - The <<_locator,locator>> to find an element.
+* `$resourceNameOrFilePath` - relative path to the file to be uploaded
+
+.Upload file_for_upload.png file
+[source,gherkin]
+----
+When I select element located `By.id(uploadfile)` and upload file `/folder/file_for_upload.png`
+----
+
+=== Verify page is not refreshed after click on element
+
+This step clicks on the element and checks that the page has not been refreshed after clicking.
+
+[source,gherkin]
+----
+When I click on element located by `$locator` then page does not refresh
+----
+
+_Deprecated syntax (will be removed in VIVIDUS 0.7.0)_:
+[source,gherkin]
+----
+When I click on an element '$locator' then the page does not refresh
+----
+
+* `$locator` - The <<_locator,locator>> to find an element to click.
+
+.Step example
+[source,gherkin]
+----
+When I click on element located by `id(send)` then page does not refresh
+----
+
+=== Verify page is not refreshed after click on element
+
+This step clicks on the element and checks that page has not been refreshed after click.
+
+[source,gherkin]
+----
+When I click on element located by `$locator` then page does not refresh
+----
+
+_Deprecated syntax (will be removed in VIVIDUS 0.7.0)_:
+[source,gherkin]
+----
+When I click on an element '$locator' then the page does not refresh
+----
+
+* `$locator` - The <<_locator,locator>> to find an element to click.
+
+.Step example
+[source,gherkin]
+----
+When I click on element located by `id(send)` then page does not refresh
+----
+
+=== Right-click on the element
+
+This step performs right-click on the element by locator.
+
+[source,gherkin]
+----
+When I perform right-click on element located by `$locator`
```

Comment:
```suggestion
When I perform right click on element located by `$locator`
```

Reply:
'right-click' is a common term according to https://dictionary.cambridge.org/dictionary/english/right-click